server/src/internalClusterTest/java/org/opensearch/action/admin/ClientTimeoutIT.java

```diff
@@ -51,6 +51,11 @@ protected Collection<Class<? extends Plugin>> nodePlugins() {
         return Collections.singletonList(MockTransportService.TestPlugin.class);
     }
 
+    @Override
```

Comment:
I don't think we want to change the behavior of non SR tests, can you pls move the SR specific tests to a new class or move to SegmentReplicationStatsIT?